mediaqueries-5/Overview.bs

```diff
@@ -688,6 +688,56 @@ Detecting a forced color palette: the 'forced-colors' feature</h3>
 			See [[!css-color-adjust-1#forced]] for details.
 	</dl>
 
+<h3 id="prefers-reduced-data">
+Detecting the desire for reduced data usage when loading a page: the 'prefers-reduced-data' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: prefers-reduced-data
+	Value: no-preference | reduce
+	For: @media
+	Type: discrete
+	</pre>
+
+	The 'prefers-reduced-data' media feature is used to detect is an user
+	has a preference for being served alternate content that uses less 
+	data for the page to be rendered.
+
+	<dl dfn-type=value dfn-for="@media/prefers-reduced-data">
+		<dt><dfn>no-preference</dfn>
+		<dd>
+			Indicates that the user has made no preference known
+			to the system. This keyword value evaluates as false
+			in the <a>boolean context</a>.
+
+		<dt><dfn>reduce</dfn>
+		<dd>
+			Indicates that user has expressed the preference for
+			lightweight alternate content.
+	</dl>
+	
+	The method by which the user expresses their preference can vary.
+	It might be a system-wide setting exposed by the Operating System,
+	or a setting controlled by the User Agent.
+	
+	<div class="example">
+		For example, a site could honour the preference of a user who 
+		has turned on data-saving mode by serving a smaller image.
+
+		<pre>
+		.image {
+			background-image: url("image/heavy.jpg");
+		}
+
+		@media (prefers-reduced-data: reduce) { 
+			/* Save-Data: On */
+			.image {
+				background-image: url("image/light.webp");
```

Comment:
There can be a future where we'd have user agents that support this MQ but not webp, so I would caution against coupling "lightweight" with browser support for formats. Having a way to do client side format negotiation for BG images would be a separate (and potentially worthwhile) use case IMO.

Reply:
Should I switch out this use-case with another one from the thread? Or change the term "lightweight"?

Reply:
I'd just switch out the "webp" extension in favour of a "jpg"

Reply:
Makes sense. Let me update that.